main/resources/com/google/refine/grel/ControlEvalError.properties

```diff
@@ -13,4 +13,4 @@ expects_second_arg_var_name={0} expects second argument to be a variable name
 expects_second_arg_index_var_name={0} expects second argument to be the index's variable name
 expects_third_arg_element_var_name={0} expects second argument to be the element's variable name
 expects_second_third_args_different={0} expects the second and third arguments to be different variable names
-expects_array_arg={0} is not an array
\ No newline at end of file
+expects_array_arg={0} expects an array as its first argument
```

Comment:
For consistency with other properties here I'd suggest naming this `expects_first_arg_array` (in case later need to have a similar constraint on a second or third argument etc.)

Reply:
Updated key